src/cascadia/TerminalApp/App.cpp

```diff
@@ -220,6 +220,25 @@ namespace winrt::TerminalApp::implementation
             const auto& profile = _settings->GetProfiles()[profileIndex];
             auto profileMenuItem = Controls::MenuFlyoutItem{};
 
+            // add the keyboard shortcuts for the first 10 profiles
+            if (profileIndex < 10)
+            {
+                auto profileShortcut = Windows::UI::Xaml::Input::KeyboardAccelerator{};

```

Comment:
Is it possible to get the actual KeyModifiers and Vkey from the Keybindings, instead of assuming they're Ctrl+Shift+<n>? 

I'm working on #537 currently, and when that merges, it's possible those won't be the actual bindings anymore.

You'd probably need to add code to AppKeyBindings to query what the KeyChord for a given ShortcutAction is

Reply:
@zadjii-msft -- is there a branch I can see how you were doing this?  I had made an assumption that there would be some keybindings that wouldn't change...specifically the profile tabs and shortcuts in the menu flyout...but if ALL potential shortcuts will be customizable, then yes, I need to see how you're planning on exposing the keybinding combination...

Reply:
Woops, my bad. I should have looked at the full code. I didn't realize we are here in a for loop.

Reply:
Sure yea - check out [dev/migrie/f/keybindings](https://github.com/microsoft/terminal/tree/dev/migrie/f/keybindings).

The plan is to have _all_ keybindings remappable. It's the only way to make sure that if a default steps on some application's toes, the user can change them to fit their needs.

Reply:
@zadjii-msft It looks like when you load the settings that the keybindings will be in memory then at that point.  Can you expose a method to query for a specific binding rather than have others re-load settings/parse/etc. since it is already there?  A static GetKeyBinding(command)?

Reply:
Yea, you could(should) definitely expose that method on AppKeyBindings. Parsing them again would be insane :P

I'd rather you do it in this PR, rather than blocking your PR on mine getting in

Reply:
Got it...a bit chicken and egg though...I need yours to be complete so I can do that reliably, no?

Reply:
I don't believe so. I think all the info you need is already in AppKeyBindings

If I had to write it here on GH, it'd look like
```c++
KeyChord AppKeyBindings::GetKeyBinding(ShortcutAction action)
{
    for (auto& kv : _keyShortcuts)
    {
        if (kv.second == action) return kv.first;
    }
    return { nullptr };
}
```

But I'm really not sure that works